lib/jsonapi-response.js

```diff
@@ -32,7 +26,7 @@ function getAttributes (data) {
   return attributes;
 }
 
-function getRelationships (data) {
+function getRelationships (data, response, config) {
```

Comment:
`response` and `config` not used (config only used to be passed to `getCreationDetails` but it is unused here too)